packages/mobile/src/screens/sign-on-screen/screens/SignOnScreen.tsx

```diff
@@ -80,21 +81,14 @@ const Background = () => {
       })}
     >
       <RadialGradient
-        style={css({
-          // NOTE: Width/Height styles are mandatory for gradient to work. Otherwise it will crash the whole app 🫠
```

Comment:
were height and width not necessary? did the comment lie?

Reply:
it is for the straight-up radial gradient, but not for our special one 🥳